Emojigrams/Emojigrams/EmojigramGuesserViewControllerViewController.swift

```diff
@@ -38,6 +39,13 @@ class EmojigramGuesserViewController: EmojigramGuesserBaseViewController {
         self.updatePage(with: emojigram)
     }
     
+    func configurePage() {
+        repeat {
```

Comment:
#pp awesome to see a repeat while block , its been a while :)

Reply:
True, it'd be good if it did anything though :P It's a ` repeat while(false)` so the loop will happen exactly once, which means there's no good purpose for having a loop in the first place. 

Fun fact: there a couple places in the C by GE code base that has a do-while(0) that has the same "purpose".